Web Inspector: Remove instrumentation for CSS Grid Overlay diagnostics

rcaliman-apple opened the pull request:
#### bd7d0c2748166af9f6f63948e565875466e9b810
<pre>
Web Inspector: Remove instrumentation for CSS Grid Overlay diagnostics
<a href="https://bugs.webkit.org/show_bug.cgi?id=242436">https://bugs.webkit.org/show_bug.cgi?id=242436</a>

Reviewed by Devin Rousso and Patrick Angle.

We&apos;ve learned what we wanted to know about preferred ways to launch the CSS Grid Overlay and popular configuration options.
Removing the diagnostics instrumentation because there&apos;s no need to collect additional data.

* Source/WebInspectorUI/UserInterface/Base/Main.js:
(WI.contentLoaded):
* Source/WebInspectorUI/UserInterface/Controllers/GridOverlayConfigurationDiagnosticEventRecorder.js: Removed.
* Source/WebInspectorUI/UserInterface/Controllers/GridOverlayDiagnosticEventRecorder.js: Removed.
* Source/WebInspectorUI/UserInterface/Main.html:
* Source/WebInspectorUI/UserInterface/Models/DOMNode.js:
(WI.DOMNode.prototype.showLayoutOverlay):
* Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js:
(WI.DOMTreeElement.prototype._layoutBadgeClicked):
* Source/WebInspectorUI/UserInterface/Views/NodeOverlayListSection.js:
(WI.NodeOverlayListSection.prototype.layout):

Canonical link: <a href="https://commits.webkit.org/252422@main">https://commits.webkit.org/252422@main</a>
</pre>

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js:
```diff
@@ -2057,9 +2057,7 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
         if (this.representedObject.layoutOverlayShowing)
             this.representedObject.hideLayoutOverlay();
         else {
```
Style: we can drop the `{` `}` now too :)

rcaliman-apple committed:
Web Inspector: Remove instrumentation for CSS Grid Overlay diagnostics
https://bugs.webkit.org/show_bug.cgi?id=242436

Reviewed by Devin Rousso and Patrick Angle.

We've learned what we wanted to know about preferred ways to launch the CSS Grid Overlay and popular configuration options.
Removing the diagnostics instrumentation because there's no need to collect additional data.

* Source/WebInspectorUI/UserInterface/Base/Main.js:
(WI.contentLoaded):
* Source/WebInspectorUI/UserInterface/Controllers/GridOverlayConfigurationDiagnosticEventRecorder.js: Removed.
* Source/WebInspectorUI/UserInterface/Controllers/GridOverlayDiagnosticEventRecorder.js: Removed.
* Source/WebInspectorUI/UserInterface/Main.html:
* Source/WebInspectorUI/UserInterface/Models/DOMNode.js:
(WI.DOMNode.prototype.showLayoutOverlay):
* Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js:
(WI.DOMTreeElement.prototype._layoutBadgeClicked):
* Source/WebInspectorUI/UserInterface/Views/NodeOverlayListSection.js:
(WI.NodeOverlayListSection.prototype.layout):

Canonical link: https://commits.webkit.org/252422@main
